src/components/SponsorsSection/styles.jsx

```diff
@@ -46,3 +46,16 @@ export const SponsorIcon = ({ pic, alt, size, link }) => (
     </a>
   </SponsorLogo>
 );
+
+export const Alpha = styled.div`
```

Comment:
Heyy @padmajabhol, it would be great if you could just change the name of this styled component to something else like `Wrapper` for example. `Alpha` as the name of this component might be a bit misleading. Rest everything LGTM 🚀

Reply:
Forgot to rename it.